boards/arm/rpi_pico/Kconfig.defconfig

```diff
@@ -1,10 +1,11 @@
 # Copyright (c) 2021 Yonatan Schachter
 # SPDX-License-Identifier: Apache-2.0
 
-if BOARD_RPI_PICO
+if BOARD_RPI_PICO || BOARD_RPI_PICO_W
 
 config BOARD
-	default "rpi_pico"
+	default "rpi_pico" if BOARD_RPI_PICO
+	default "rpi_pico_w" if BOARD_RPI_PICO_W	
```

Comment:
There is an unwanted trailing white space.

Reply:
Removed the extra whitespace.